pkgs/development/python-modules/spacy/default.nix

```diff
@@ -56,16 +56,16 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.cfg \
-      --replace "blis>=0.4.0,<0.5.0" "blis>=0.4.0,<1.0" \
+      --replace "blis>=0.4.0,<0.8.0" "blis>=0.4.0,<1.0" \
       --replace "catalogue>=0.0.7,<1.1.0" "catalogue>=0.0.7,<3.0" \
       --replace "plac>=0.9.6,<1.2.0" "plac>=0.9.6,<2.0" \
       --replace "srsly>=1.0.2,<1.1.0" "srsly>=1.0.2,<3.0" \
-      --replace "thinc==7.4.1" "thinc>=7.4.1,<8"
+      --replace "thinc>=7.4.1,<7.5.0" "thinc>=7.4.1,<8"
```

Comment:
If you're relaxing dependencies, would it make sense to just change them to e.g. ``blis`` vs ``blis>=0.4.0...``?

Reply:
I can see the arguments on both sides. But as a maintainer, I'd prefer to check compatibility upstream if thinc (or any of the other dependencies) has a new major version, than possibly silently failing (since we cannot run the regular unit tests and instead rely on the single `passthru.tests` test).

Moreover, thinc should not be updated without taking spaCy into consideration anyway. spaCy is the only downstream package that uses it, so thinc is in nixpkgs for spaCy.